src/script/components/resource-hub-cards.ts

```diff
@@ -7,55 +7,62 @@ interface CardData {
   linkUrl: string;
 }
 
-const resourceCards: Array<CardData> = [{
-  imageUrl: "/assets/icons/icon_120.png",
-  title: "Blog",
-  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
-  linkUrl: ""
-}, {
-  imageUrl: "/assets/icons/icon_120.png",
-  title: "Demo",
-  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
-  linkUrl: ""
-}, {
-  imageUrl: "/assets/icons/icon_120.png",
-  title: "Components",
-  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
-  linkUrl: ""
-}, {
-  imageUrl: "/assets/icons/icon_120.png",
-  title: "Documentation",
-  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
-  linkUrl: ""
-}]
+const resourceCards: Array<CardData> = [
+  {
+    imageUrl: '/assets/icons/icon_120.png',
+    title: 'Blog',
+    description:
+      'Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing',
+    linkUrl: '',
+  },
+  {
+    imageUrl: '/assets/icons/icon_120.png',
+    title: 'Demo',
+    description:
+      'Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing',
+    linkUrl: '',
+  },
+  {
+    imageUrl: '/assets/icons/icon_120.png',
+    title: 'Components',
+    description:
+      'Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing',
+    linkUrl: '',
+  },
+  {
+    imageUrl: '/assets/icons/icon_120.png',
+    title: 'Documentation',
+    description:
+      'Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing',
+    linkUrl: '',
+  },
+];
 
 // for the landing page
 export function landingCards() {
-  return resourceCards.map(data => {
-    return renderResourceCard(data)
-  })
+  return resourceCards.map((data) => {
+    return renderResourceCard(data);
+  });
 }
 
 // For the complete page
 export function completeCards() {
-  return resourceCards.slice(1).map(data => {
-    return renderResourceCard(data)
-  })
+  return resourceCards.slice(1).map((data) => {
```

Comment:
Nit: not a fan of adding the optional parens in this case. Is this a rule in our linter? If so, can we change it?

Reply:
oh yeah, it totally is, I'll add the change here

Reply:
https://github.com/pwa-builder/PWABuilder/pull/1359/commits/403e7578b694755e7e10d5e578d184838ab4f508